src/components/Documentation/ApiGuide/CopyText.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styles from "./ApiGuideContent.module.scss";
+import Image from "next/image";
+
+type  CopyTextProps = {
+    oauth: string
+}
+
+const CopyText: React.FC<CopyTextProps> = ({ oauth }) => {
+    return (
+        <div className={styles["card-light"]}>
+            <text className={styles["card-light-content"]}>
+                {oauth}
+            </text>
+            <div className={styles["copy-button"]}>
+                <Image
+                    className={styles["copy-button-image"]}
+                    src="/copy.svg"
+                    width="22"
+                    height="22"
+                    alt="copy code icon"
+                />
+                <text onClick={() => {navigator.clipboard.writeText(oauth)}}>Copy</text>
```

Comment:
navigator.clipboard.writeText is not supported in IE. Please have a look.

Also this component is required in quickstart codesample component. If both have same style, we can reuse this component there after merging.

Reply:
Created separated task for this, will be updated by using CopyButton component.